livelog/livelog.go

```diff
@@ -97,13 +103,25 @@ func (l *LiveLog) connectInputStream() error {
 	}
 	client := new(http.Client)
 	go func() {
```

Comment:
Why are we running this in a go routine? Is it so the task starts and continues on while this waits for the live log to start?